src/xrpld/app/tx/detail/Transactor.cpp

```diff
@@ -145,18 +146,6 @@ preflight2(PreflightContext const& ctx)
 
 //------------------------------------------------------------------------------
 
-PreflightContext::PreflightContext(
```

Comment:
Need a check in `preflight1` that ensures that you can't have transactions with the `tfInnerBatchTxn` flag without the `Batch` amendment enabled.

Also probably a check in `Change` to ensure that that flag can't be on pseudo-transactions at all.